### DISCUSSION TITLE
Will this course replace Dive into Deep Learning Compiler?

### DISCUSSION THREAD

**Discussion by shanmo** [mlc-ai/mlc-en#5] [General]
Hi, will this course replace [Dive into Deep Learning Compiler](https://tvm.d2l.ai/index.html)? `tvm.d2l.ai` is not being updated ATM.

**Comment by Hzfengsy** [comment]
We do not intentionally replace any existing course. On the contrary, we are happy to see more tutorials or courses for the MLC topic.
